Tests/Route/DefaultRouteGeneratorTest.php

```diff
@@ -51,6 +51,10 @@ public function testGenerateUrl($expected, $name, array $parameters)
         $collection->add('foo');
         $collection->addCollection($childCollection);
 
+        $nochildCollection = new RouteCollection('base.Code.Child', 'admin_child', '/foo/', 'BundleName:ControllerName');
```

Comment:
> You're not a friend of long lines :smile:

I use vim and I always have split screen with most of the times 4 files opened. This is the single most important improvement to my workflow in terms of speed : no more looking for the correct tab in a huge list of files I have opened in the past when browsing my project. Have been a fierce enemy of long lines since then ;)